espanol/canudos.md

```diff
@@ -100,3 +100,4 @@ Salió corriendo y no se lo vio nunca más en el reino
 
 Conectó su guitarra al amplificador y se puso a tocar una versión de "Siempre es lo mismo, nena", pero sin distorsión
 
+[Pero luego se quedo cansado y se fue a dormir](cansado/cansado.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.